apps/demos/Demos/DropDownButton/Overview/React/styles.css

```diff
@@ -16,3 +16,55 @@
   cursor: pointer;
   font-size: 18px;
 }
+
+.dx-button-wrapper {
+  padding-inline-start: 12px;
+  padding-inline-end: 4px;
+  display: flex;
+  align-items: flex-start;
+  flex-direction: column;
+}
+
+.button-img-wrapper {
+  position: relative;
+  height: 32px;
+}
+
+.dx-button-img {
+  width: 32px;
+  height: 32px;
+  position: relative;
+  border-width: 1px;
+  border-style: solid;
+  border-color: var(--dx-color-border);
+  border-radius: 50%;
+}
+
+.button-img-indicator {
+  position: absolute;
+  background-color: var(--dx-color-danger);
+  top: -1px;
+  right: -1px;
+  width: 10px;
+  height: 10px;
+  border-radius: 50%;
+  border-width: 2px;
+  border-style: solid;
+  border-color: var(--dx-color-main-bg);
+  z-index: 1;
+}
+
+.dx-button-title {
+  line-height: 20px;
+}
+
+.dx-button-row {
+  font-size: 12px;
+  line-height: 14px;
+  opacity: 0.6;
+}
+
+#two-sections .dx-button {
```

Comment:
i suppose we can leace selector as #two-sections ( without .dx-button). But please check it

Reply:
.dx-button is needed to resize to button when other themes are used